### PR TITLE
Added a HTML question

### PR DESCRIPTION
**Pull request by ARYAN-CODES-STAR**
Checklist:

- [ ] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [ ] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/html-quiz.ts [hunk 1]
```diff
@@ -1649,5 +1649,16 @@ const htmlQuiz = [
       "Hypertext in HTML refers to text (often with embeds such as images, too) that is organized in order to connect web pages to one another.",
     Link: "https://www.freecodecamp.org/news/what-is-html-definition-and-meaning/"
   }
+  {
+  Question:
+  "In HTML, which attribute is used to increase speed of marquee? ",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1654]:
```suggestion
  "In HTML, which attribute is used to increase the speed for the deprecated marquee tag? ",
```

File: src/data/html-quiz.ts [hunk 2]
```diff
@@ -1649,5 +1649,16 @@ const htmlQuiz = [
...
+Answer: "scrollamount",
+Distractor1:"speed",
+Distractor2:"markspeed",
+Distractor3:"scrollmount",
+  Explanation:
+" The Marquee scrollamount attribute in HTML is used to set the amount of scrolling at each interval in pixels.",
+Link: "https://www.geeksforgeeks.org/html-marquee-scrollamount-attribute/  ",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1661]: Please only user MDN or freeCodeCamp resources

 ```suggestion
Link: "https://developer.mozilla.org/en-US/docs/Web/HTML/Element/marquee#attr-scrollamount",
```

File: src/data/html-quiz.ts [hunk 3]
```diff
@@ -1649,5 +1649,16 @@ const htmlQuiz = [
       "Hypertext in HTML refers to text (often with embeds such as images, too) that is organized in order to connect web pages to one another.",
     Link: "https://www.freecodecamp.org/news/what-is-html-definition-and-meaning/"
   }
+  {
+  Question:
+  "In HTML, which attribute is used to increase speed of marquee? ",
+Answer: "scrollamount",
+Distractor1:"speed",
+Distractor2:"markspeed",
+Distractor3:"scrollmount",
```
Comment (jdwilkin4) [inline src/data/html-quiz.ts:1658]: Please fix the formatting here so it matches the other questions

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/html-quiz.ts (hunk 1 above)
Comment (ARYAN-CODES-STAR) [reply src/data/html-quiz.ts]: DONE

File: src/data/html-quiz.ts (hunk 3 above)
Comment (ARYAN-CODES-STAR) [reply src/data/html-quiz.ts:1658]: Yeah issue resolved

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**